docker-stack.yml

```diff
@@ -80,10 +85,9 @@ services:
                 if [ $$django_tables_exist -gt 0 ] || [ $$REFRESH_DB == 'true' ]; then
                     ./refresh-data.sh 
                     ./cfgov/manage.py update_index
-                    httpd -d cfgov/apache -D FOREGROUND
-                else
-                    httpd -d cfgov/apache -D FOREGROUND
-                fi"
+                fi
+                
+                httpd -d cfgov/apache -D FOREGROUND"
```

Comment:
Saw some cleanup could be done to this, we don't really need an else block if its the same as the bottom of the if block.